### DISCUSSION TITLE
[Office Hours]

### DISCUSSION THREAD

**Discussion by kcallon** [CS342/.github#57] [Office Hours]
### Related Discussion

![image0](https://github.com/CS342/.github/assets/70660419/978274d8-5c43-4558-9638-ece5b799f7e7)

### Reproduction

We're currently struggling with merge conflicts because this branch is 1 commit behind main and 2 commits ahead. 

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by philippzagar** [comment]
Hi @kcallon, thanks for creating a GitHub Issue for the office hours! As far as I see from your screenshot, the conflicts occur in the Xcode project files (sadly a pretty common source for merge conflicts). Resolving the conflicts may be difficult and the screenshot above provides too little context: Let's discuss the details in one of the office hours this week!

**Reply by PSchmiedmayer** [reply]
@kcallon thank you for sharing your issues here in GitHub Discussions! You were not the only one having these issues and it is great to build up a knowledge repository around these issues!

I hope this issue has already been resolved. If so, can you and @philippzagar share what the solution was in the office hours so anyone who stumbles across the same issue can follow your solution ideas?

**Reply by philippzagar** [reply]
@kcallon I cannot remember fixing this issue with you during office hours. May you be able to share the solution you came up with?
